platform/src/main/java/org/stellar/anchor/platform/paymentobserver/StellarTransaction.java

```diff
@@ -0,0 +1,23 @@
+package org.stellar.anchor.platform.paymentobserver;
+
+import com.google.gson.annotations.SerializedName;
+import java.time.Instant;
+import lombok.Builder;
+import lombok.Data;
+import org.stellar.anchor.paymentservice.Payment;
+
+@Data
+@Builder
+public class StellarTransaction {
```

Comment:
❓  Can't we use the Transaction from the Stellar SDK?

Reply:
🤔  Maybe we should send over the PaymentOperation instead, since a Transaction can contain multiple payments/operations.